WritingDashboard/src/setupProxy.js

```diff
@@ -4,7 +4,7 @@ module.exports = function(app) {
   app.use(
     '/api',
     createProxyMiddleware({
-      target: 'https://backend-lv2fs4hn6a-ew.a.run.app',
+      target: 'http://localhost:5000',
```

Comment:
target is nu localhost, is dit de bedoeling

also, http of https?

Reply:
ja